devcontainer/Dockerfile

```
@@ -0,0 +1,13 @@
+ARG SWIFT_VERSION=6.2
```

```suggestion
ARG SWIFT_VERSION=latest
```

For your consideration, as that'll reduce the maintenance burden on our many repos.